src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1495,5 +1498,40 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.since("INSERT VERSION")
 				.requiredPlugins("Paper 1.16+"));
 		}
+
+		Classes.registerClass(new ClassInfo<>(BossBar.class, "bossbar")
+			.user("boss ? bars?")
+			.name("Boss Bar")
+			.description("Represents a boss bar")
+			.since("INSERT VERSION")
+			.parser(new Parser<BossBar>() {
+				@Override
+				public boolean canParse(ParseContext context) {
+					return false;
+				}
+
+				@Override
+				public String toString(BossBar bossBar, int flags) {
+					return "bossbar with title \"" + bossBar.getTitle() + "\"";
+				}
+
+				@Override
+				public String toVariableNameString(BossBar bossBar) {
+					return toString(bossBar, 0);
+				}
```

Comment:
```suggestion
				.user("boss ? bars?")
				.name("Boss Bar")
				.description("Represents a boss bar")
				.since("INSERT VERSION")
				.parser(new Parser<BossBar>() {
					@Override
					public boolean canParse(ParseContext context) {
						return false;
					}

					@Override
					public String toString(BossBar bossBar, int flags) {
						return "bossbar with title \"" + bossBar.getTitle() + "\"";
					}

					@Override
					public String toVariableNameString(BossBar bossBar) {
						return toString(bossBar, 0);
					}
```